sdk/scala-sdk-testkit/src/main/scala/kalix/scalasdk/testkit/ActionResult.scala

```diff
@@ -48,6 +48,12 @@ trait ActionResult[T] {
    */
   def forwardedTo: DeferredCallDetails[_, T]
 
+  def singleTimerDetails: Seq[SingleTimerDetails[_, _]]
+
+  def nextSingleTimerDetails[I, O]: SingleTimerDetails[I, O]
+
+  def timerCancellations: Seq[String]
```

Comment:
`next` variant for the timer because we can ask and cast to the types from the deferred call.

`timerCancellations` are just strings so can be just iterated without extra help.